frontend/layouts/default.vue

```diff
@@ -144,6 +178,7 @@ export default {
     return {
       title_1: 'Philo',
       title_2: 'Biblon',
+      version: '0.6.0',
```

Comment:
Remove version from here. The version was moved to nuxt.config.js and is accessible via: $config.version